exercice/exercice-one.js

```diff
@@ -0,0 +1,41 @@
+// 1. Write a function(s) that checks the level of nesting of an array. Then, use that function to flatten the array into a single-dimensional array.
+
+function getNestingLevel(array) {
+  if (!Array.isArray(array)) {
+    return 0;
+  }
+
+  let maxNesting = 1;
```

Comment:
- [ ] How about reducing redundant variables 😄 how about declaring the level outside the if  and use it instead of using this masNesting variable which is not adding any value on this function 😃

Reply:
ok